### PR TITLE
OSDOCS-11520:adds support note to OVN-K migration procedure

### PR DESCRIPTION
**Pull request by JoeAldinger**
Version(s):
4.16

Issue:
https://issues.redhat.com/browse/OSDOCS-11520

Link to docs preview:
https://79795--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.html

QE review:
- [x] QE has approved this change.

Additional information:
https://github.com/openshift/openshift-docs/pull/79740 SDN removal PR

### CODE REVIEW

File: networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc [hunk 1]
```diff
@@ -8,6 +8,11 @@ toc::[]
 
 As a cluster administrator, you can migrate to the OVN-Kubernetes network plugin from the OpenShift SDN network plugin using the _offline_ migration method or the limited live migration method.
 
+[WARNING]
+====
+There is no supported migration from OpenShift SDN network plugin to OVN-Kubernetes network plugin in {product-title} 4.17. You must migrate to the OVN-Kubernetes plugin before upgrading to {product-title} 4.17.
```
Comment (dfitzmau) [inline networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc:13]:
```suggestion
No supported method exists for migrating from the OpenShift SDN network plugin to the OVN-Kubernetes network plugin in {product-title} 4.17. You must migrate to the OVN-Kubernetes plugin before upgrading to {product-title} 4.17.
```

**Review by dfitzmau — APPROVED** [review]
/remove-label peer-review-in-progress

/label peer-review-done

File: networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc (hunk 1 above)
Comment (danwinship) [inline networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc:13]: I don't think this is the right warning; it suggests that SDN users might be able to upgrade to 4.17, but then wouldn't be able migrate to OVN-Kubernetes (at least, not in a "supported" way). But the actual problem is that SDN users can't even upgrade to 4.17 (and so they can't even get to the point where they might try "migrating from SDN to OVN in 4.17").

```suggestion
It is not possible to upgrade a cluster to {product-title} 4.17 if it is using the OpenShift SDN network plugin. You must migrate to the OVN-Kubernetes plugin before upgrading to {product-title} 4.17.
```

If we put a warning here, it should be along the lines of "it's not possible to upgrade to 4.17 if you are using SDN, so you need to migrate to OVN-K before then".

**Comment by danwinship** [comment]
> Version(s):
> 4.16+

More like "4.16-" isn't it? This section of the docs will no longer exist as of 4.17.

**Comment by danwinship** [comment]
lgtm too

**Comment by JoeAldinger** [comment]
closed in favor of #80652

**Pull request closed without merging**